src/components/panel/panel.tsx

```diff
@@ -211,6 +205,7 @@ export class Panel implements ConditionalSlotComponent, InteractiveComponent {
 
   dismiss = (): void => {
     this.dismissed = true;
+    this.calcitePanelDismissedChange.emit();
```

Comment:
Sidebar: should we deprecate this in favor of a `calcitePanelDismiss` event since it no longer maps to `dismissed` changing?

Reply:
yeah that makes sense 👍